pkg/util/motd.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2023 The KEDA Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/go-logr/logr"
+
+	"github.com/kedacore/keda/v2/version"
+)
+
+const (
+	minSupportedVersion = 23
+	maxSupportedVersion = 26
+)
+
+func PrintMotd(logger logr.Logger, kubeVersion K8sVersion, component string) {
```

Comment:
Q: What does Motd stand for?

Reply:
[Message Of The Day](https://en.wikipedia.org/wiki/Message_of_the_day)

Reply:
That's why I never use abbreviations in code :)

```suggestion
func PrintMessageOfTheDay(logger logr.Logger, kubeVersion K8sVersion, component string) {
```